tests/sources/test_notebooksource.py

```diff
@@ -636,6 +636,103 @@ def test_save_injected_cell_in_paired_notebooks(tmp_nbs, prefix):
     assert get_injected_cell(jupytext.read(Path(prefix, 'load.ipynb')))
     assert get_injected_cell(jupytext.read(Path('load.py')))
 
+def test_move_script_save_injected_cell_in_paired_notebooks(tmp_nbs):
```

Comment:
test_move_script_save_injected_cell_in_paired_notebooks(tmp_nbs, injection_setup)

fixture: def injection_setup:
```
Path('scripts').mkdir(parents=True, exist_ok=True)
Path('load.py').rename('scripts/load.py')
Path('clean.py').rename('scripts/clean.py')
Path('plot.py').rename('scripts/plot.py')
```

Reply:
https://docs.pytest.org/en/6.2.x/fixture.html

Reply:
conftest

Reply:
@idomic If I use `nb-nested` should I still make this fixture? Since the files are already in nested folders, I wasn't sure if it was necessary.

Reply:
Please disregard that last message, I didn't realize this was a comment from the first commit.